data/json/monsters.json

```diff
@@ -1483,7 +1483,7 @@
     "id": "mon_gracke",
     "type": "MONSTER",
     "name": "gracken",
-    "description": "This is some form of eldritch monstrosity; an uncouth black being with smooth, oily, skin and unpleasant horns that curve inward toward each other.  Tall and thin, the shadows cling unnaturally to its vaguely defined humanoid form as it shuffles along, its hands twitching and spasming so rapidly as to appear a little more than a black blur of claws.  Gazing upon its disturbing form fills you with an unspeakable terror.",
+    "description": "You think this is some kind of bizarre humanoid; an uncouth black being with smooth, oily, skin and unpleasant horns that curve inward toward each other.  Its hands twitch and spasming so rapidly as to appear a little more than a black blur of claws. Gazing on its form fills you with a strange, indescribable kind of horror.",
```

Comment:
Missing second space before `Gazing`

Reply:
Also, 'spasming' should be changed to 'spasm' to be brought in line with the other changes.